src/redux/api/appointmentApi.ts

```diff
@@ -149,6 +150,13 @@ export const appointmentApi = createApi({
       }),
       invalidatesTags: (result, error, { id }) => [{ type: 'Appointments', id }, 'Appointments'],
     }),
+    deleteAppointment: builder.mutation<void, string>({
+      query: (id) => ({
+        url: `${route.appointment}/${id}`,
+        method: 'DELETE',
+      }),
+      invalidatesTags: (result, error, id) => [{ type: 'Appointments', id }, 'Appointments'],
```

Comment:
why do you use invalidatesTags if you have refetch?
why do you invalidate not only 'Appointments' but also { type: 'Appointments', id }?

Reply:
It is also used for appointments shown for user, that functionality is used there